packages/app/src/pages/endpoints/[endpointId]/_/content/parts/body/index.tsx

```diff
@@ -89,6 +98,11 @@ const Body: React.FC<Props> = ({
     sortState,
   ]);
 
-  return <div className={className}>{elm}</div>;
+  return (
+    <>
+      <div className={className}>{elm}</div>
+      <Error.renewal {...error.bind} withModal={true} />
```

Comment:
nits: use syntax sugar instead.

```suggestion
      <Error.renewal {...error.bind} withModal />
```